packages/node_modules/nav-frontend-ikoner-assets/assets/pil-hoyre.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 24 24">
```

Comment:
Hvorfor legges denne til? Dekker ikke chevron-komponenten dette behovet?

Reply:
Vi har behov for å bruke en .svg istedetfor en React-komponent. Chevron-komponenten sånn jeg forstår det implementerer chevron gjennom CSS.

Reply:
Ikke sikker på om jeg forstår usecasen deres, og om det absolutt må være en svg så høres det mistenkelig ut som ett prosjekt-spesifikt issue. Men vi kan ta det på møtet i morgen

Reply:
Jeg tenker egentlig at det er et nav-frontend-moduler issue om ikke ikoner-assets er en pålitelig kilde for å enkelt ta i bruk ikoner fra ikonbiblioteket, uten at man må gå via en React-komponent eller et stylesheet som i dette tilfellet implementerer ett enkeltikon "manuelt". Chevron i denne sammenheng er egentlig å betrakte som en util for å vende på ikon og endre størrelse imo. Vi bør vel være konsekvente i hvordan vi tilgjengeliggjør ikonene våre for prosjektene.

Reply:
Men ikoner-assets skal ikke være kilden for ikoner fra ikonbiblioteket. Det har jo vært sagt fra starten av. Om dette har endret på seg må ikoner-assets modulen gjøres om, slik at den faktisk støtter å ha den mengden ikoner. 

Angående Chevrons så har nok jeg en helt annen oppfatning enn deg. Det er ikke ment som en util for å vende på ikoner, men som en erstatning av tidligere pil-ikon. Man kan godt se på å få flyttet den inn i ikoner-assets om man mener det er viktig at alt kommer derifra. Men før ikon-problematikken her på huset faktisk blir løst så ser jeg ikke helt poenget.

Reply:
Jeg ser ikke helt problemet - poenget er å tilgjengeliggjøre pil-ikonet på samme måte som alle de andre ikonene som er i ikoner-assets idag blir tilgjengliggjort. Hvorfor skal man måtte tilpasse løsningene til at nav-frontend leverer ikoner på flere måter? Greit nok at man lager en chevron-komponent implementert i css for å slippe å styre med svg (eller hva som er grunnen for at det er løst med CSS), men det skal vel ikke hindre andre i å bruke det som .svg uten å trenge å ta inn ikonet i sitt eget prosjekt.

Reply:
Mener chevrons ble implementert sånn pga animasjoner. Og jo, det bør så absolutt hindre folk fra å bruke en svg-variant av pilen imho. Det har hittil aldri vært behov for en svg-variant, og det å introdusere en svg-variant av en allerede eksisterende komponent gjør bare at vi må forvalte to versjoner, og passe på at alle design endringer blir propagert til begge to.

Om man vil ha svg-varianten bør tre ting skje; må få inn de tre andre retningen av pilen, skrive eksisterende kompontener bort fra chevron-komponenten, og til slutt slette den modulen. Det å vedlikeholde to versjoner av "samme" komponent fremover virker for meg veldig unaturlig.

Reply:
Vi kan ikke slette Chevron-komponenten om den er implementert sånn med tanke på animasjoner, som for øvrig er et annet behov enn behovet for å ta i bruk dette ikonet på lik måte som man tar i bruk alle andre ikoner fra nav-frontend. Om noe, burde det vel være Chevron-komponenten som skrives om til å bruke ikoner fra ikoner-assets istedenfor at den skal tegne det opp selv (om det da går an å bevare animasjonene med det), slik som andre moduler gjør når de skal ta i bruk ikoner.

Ved å si at ikonene tilgjengeliggjøres på X forskjellige måter i nav-frontend for X forskjellige behov og si at prosjektene bare må tilpasse seg deretter, gjør man hele den delen av nav-frontend uforutsigbar og vanskelig å forholde seg til. Da er det mye bedre å si at ikoner i .svg-filer ligger samlet på ett sted, og la komponenter som implementerer animasjoner for transisjoner mellom f.eks. forskjellige orientasjoner i runtime'n være noe eget.